charts/velero/README.md

```diff
@@ -24,6 +24,30 @@ The default configuration values for this chart are listed in values.yaml.
 
 See Velero's full [official documentation](https://velero.io/docs/v1.4/basic-install/). More specifically, find your provider in the Velero list of [supported providers](https://velero.io/docs/v1.4/supported-providers/) for specific configuration information and examples.
 
+#### Configuration
```

Comment:
I will take a pass at this tomorrow. For now: we had in the past concluded that it was best to keep all the parameter listing and documentation in the values.yaml file only, so as to not have to maintain any duplication in the readme. I still think that is best.